warehouse/query-core/src/main/java/datawave/query/jexl/lookups/FieldNameIndexLookup.java

```diff
@@ -117,7 +124,10 @@ public synchronized IndexLookupMap lookup() {
         if (!sessions.isEmpty()) {
             try {
                 // for field name lookups, we wait indefinitely
-                timedScanWait(timedScanFuture, lookupStartedLatch, lookupStoppedLatch, lookupStartTimeMillis, Long.MAX_VALUE);
+                // TODO consider if this really should be Long.MAX_VALUE or some time less. Other index scanners are set to config.getMaxIndexScanTimeMillis().
```

Comment:
Do you need this TODO if you are now passing config.getMaxAnyFieldScanTimeMillis?

Reply:
If you are referring to the default value of the timeout, then we could change the default to be that of the query timeout.

Reply:
I wanted the TODO so we talk about what the value should be. Yes it's now configurable. But if no one looks at it it won't ever change.